sherpa/astro/ui/tests/test_astro_ui.py

```diff
@@ -428,7 +429,11 @@ def save_arrays(self):
         rtol = 0
         atol = 1e-5
         self.assertIsInstance(out, Data1D)
-        self.assertEqual(out.name, ofh.name, msg="file name")
+
+        # remove potential dm syntax introduced by backend before checking for equality
+        out_name = re.sub("\[.*\]", "", out.name)
```

Comment:
I had the same thought right after I pushed the commit, and then I thought it wasn't worth being changed because if the backend didn't happen to append the dm syntax then other parts of the test would fail.